### PR TITLE
Fixed SBR request in route_k8s_resource example, by setting group of applicationSelector and fixing whitespaces for easier copy/pasting. #526

### PR DESCRIPTION
**Pull request by noelo**
### Motivation

Readme gives malformed json payload and incorrect applicationSelector

### Changes

Fixed SBR request in route_k8s_resource example, by setting group of applicationSelector and fixing whitespaces for easier copy/pasting.

### Testing

<< How to test those changes, reference implementation of unit/e2e tests. >>

For further more details refer the CONTRIBUTING.md

### CODE REVIEW

**Comment by Avni-Sharma** [comment]
Thanks @noelo 
/lgtm
/approve

File: examples/route_k8s_resource/README.md [hunk 1]
```diff
@@ -99,22 +99,22 @@ cat <<EOS |kubectl apply -f -
...
+metadata:
   name: binding-request
   namespace: service-binding-demo
 
-spec: 
-  applicationSelector: 
-    group: ""
+spec:
+  applicationSelector:
+    group: apps
```
Comment (pedjak) [inline examples/route_k8s_resource/README.md:108]: @noelo if I understand correctly, this line is only real change in PR? The rest of changes are whitespace formatting?

**Comment by noelo** [comment]
yep the whitespace is needed to enable people to copy/paste the yaml

**Comment by noelo** [comment]
Updated as requested

**Comment by Avni-Sharma** [comment]
Hi @noelo Can you rephrase your commit message https://github.com/redhat-developer/service-binding-operator/pull/527/commits/f76039b8cb3f2f180358c56220572c53c4dc9397 as suggested by @pedjak  so that we can get your PR merged. Thanks :D
